Task08/Task08/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Task_08
+{
+    internal class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Введите значение x:");
+            double x = double.Parse(Console.ReadLine());
+            Console.WriteLine("f(x) = " + yFromX(x));
+
+            Console.ReadKey();
+        }
+        static double yFromX(double x)
+        {
+            if (x < -2) { return -2; }
+            else if (x > 2) { return 2; }
+            else { return x; }
+        }
```

Comment:
1. Разделяйте методы в классе пустой строкой, так код читается лучше.
2. Лучше писать структурированные операторы условия. У вас строк меньше, а код читается хуже.
3. Если выполнение операторов внутри `if` заканчивается `retrun`, то дальнейший `else` не нужен.
4. Имя метода следует начинать с большой буквы.
```suggestion
        }
        static double YFromX(double x)
        {
            if (x < -2)
                return -2;
            
            if (x > 2)
                 return 2;
                 
            return x;
        }
```